composer.json

```diff
@@ -24,32 +24,32 @@
         }
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^7.3 || ^8.0",
         "composer-runtime-api": "^2",
-        "doctrine/dbal": "^2.11 || ^3.0",
+        "doctrine/dbal": "^3.0",
```

Comment:
We need to require `^3.3` here because you've dropped some compat code that we would still need for DBAL 3.2 and below, like checking for `getNativeConnection()`.